osquery/tables/system/darwin/processes.cpp

```diff
@@ -246,98 +345,55 @@ static inline long getUptimeInUSec() {
               tv.tv_usec);
 }
 
-QueryData genProcesses(QueryContext& context) {
-  QueryData results;
-
-  // Initialize time conversions.
-  static mach_timebase_info_data_t time_base;
-  if (time_base.denom == 0) {
-    mach_timebase_info(&time_base);
+void genProcRUsage(const QueryContext& context, int pid, Row& r) {
+  if (!context.isAnyColumnUsed({"name",
```

Comment:
Why are there fields in here that you don't look at or use? Like `name`

Reply:
Good catch. Left over from a previous attempt that got weird. ;-) Will fix.